### PR TITLE
Changes for some more of the initital setup of the start

### PR DESCRIPTION
**Pull request by JawadJ**

### CODE REVIEW

File: src/main/java/com/stefancooper/SpigotUHC/ConfigParser.java [hunk 1]
```diff
@@ -37,6 +54,21 @@ public void executeConfigurable(Configurable<?> configurable) {
...
+                break;
+            case WORLD_BORDER_CENTER_Y:
+                Double newWorldBorderY = (Double) configurable.value();
+                Double worldCenterY = Bukkit.getWorld(config.getProp(WORLD_NAME.configName)).getWorldBorder().getCenter().getY();
+                newWorldBorderY = worldCenterY;
+                break;
+            case WORLD_BORDER_CENTER_Z:
+                Double newWorldBorderZ = (Double) configurable.value();
+                Double worldCenterZ = Bukkit.getWorld(config.getProp(WORLD_NAME.configName)).getWorldBorder().getCenter().getZ();
+                newWorldBorderZ = worldCenterZ;
```
Comment (stefan-cooper) [inline src/main/java/com/stefancooper/SpigotUHC/ConfigParser.java:70]:
```suggestion
                Double newWorldCenterZ = (Double) configurable.value();
                Double worldCenterX = Bukkit.getWorld(config.getProp(WORLD_NAME.configName)).getWorldBorder().getCenter().getX();
             Bukkit.getWorld(config.getProp(WORLD_NAME.configName)).getWorldBorder().setCenter(worldCenterX, newWorldCenterZ);

```

File: src/main/java/com/stefancooper/SpigotUHC/ConfigParser.java [hunk 2]
```diff
@@ -37,6 +54,21 @@ public void executeConfigurable(Configurable<?> configurable) {
                 WorldBorder worldBorder = Bukkit.getWorld(config.getProp(WORLD_NAME.configName)).getWorldBorder();
                 worldBorder.setSize(newWorldBorderSize);
                 break;
+            case WORLD_BORDER_CENTER_X:
+                Double newWorldBorderX = (Double) configurable.value();
+                Double worldCenterX = Bukkit.getWorld(config.getProp(WORLD_NAME.configName)).getWorldBorder().getCenter().getX();
+                newWorldBorderX = worldCenterX;
```
Comment (stefan-cooper) [inline src/main/java/com/stefancooper/SpigotUHC/ConfigParser.java:60]:
```suggestion
                Double newWorldBorderX = (Double) configurable.value();
                Double worldCenterX = Bukkit.getWorld(config.getProp(WORLD_NAME.configName)).getWorldBorder().getCenter().getX();
                Bukkit.getWorld(config.getProp(WORLD_NAME.configName)).getWorldBorder().setCenter(newWorldCenterX, worldCenterZ);

```

**Review by stefan-cooper — CHANGES_REQUESTED** [review]
2 comments because right now i don't think it is working as you expect

File: src/main/java/com/stefancooper/SpigotUHC/resources/ConfigKey.java [hunk 3]
```diff
@@ -1,7 +1,26 @@
 package com.stefancooper.SpigotUHC.resources;
 
 public enum ConfigKey {
+    //World Border Enums
     WORLD_BORDER_INITIAL_SIZE("world.border.initial.size"),
+    WORLD_BORDER_FINAL_SIZE("world.border.final.size"),
+    WORLD_BORDER_SHRINKING_PERIOD("world.border.shrinking.period"),
+    WORLD_BORDER_GRACE_PERIOD("world.border.grace.period"),
+    WORLD_BORDER_CENTER_X("world.border.center.x"),
+    WORLD_BORDER_CENTER_Y("world.border.center.y"),
```
Comment (stefan-cooper) [inline src/main/java/com/stefancooper/SpigotUHC/resources/ConfigKey.java:10]: i also don't know if we need a world center y, cos you can't set that in the spigot api

File: src/main/java/com/stefancooper/SpigotUHC/ConfigParser.java (hunk 1 above)
Comment (JawadJ) [reply src/main/java/com/stefancooper/SpigotUHC/ConfigParser.java]: dun

File: src/main/java/com/stefancooper/SpigotUHC/ConfigParser.java (hunk 2 above)
Comment (JawadJ) [reply src/main/java/com/stefancooper/SpigotUHC/ConfigParser.java]: dun

File: src/main/java/com/stefancooper/SpigotUHC/resources/ConfigKey.java (hunk 3 above)
Comment (JawadJ) [reply src/main/java/com/stefancooper/SpigotUHC/resources/ConfigKey.java]: removed

**Review by stefan-cooper — APPROVED** [review]
LGTM

**Pull request merged**